src/main/cpp/sequencer.cpp

```diff
@@ -167,10 +181,17 @@ void Sequencer::collectSequencedEvents( BaseInstrument* instrument, int bufferPo
             if (( eventStart >= bufferPosition && eventStart <= bufferEnd ) ||
                 ( eventStart <  bufferPosition && eventEnd >= bufferPosition ))
             {
-                if ( !audioEvent->isDeletable())
+                if ( !audioEvent->isDeletable()) {
+                    if ( checkForDuplicates && EventUtility::vectorContainsEvent( channel->audioEvents, audioEvent )) {
+                        continue;
```

Comment:
Addresses issue that could occur when sequences would loop and events would overlap into the looped region (these would be rendered twice which would create an audible volume spike for the length of the loop offset overlap (a fraction of a single buffer)).